src/snowflake/snowpark/_internal/analyzer/snowflake_plan_node.py

```diff
@@ -39,12 +71,28 @@ def __init__(self, start: int, end: int, step: int, num_slices: int = 1) -> None
         self.step = step
         self.num_slices = num_slices
 
+    @property
+    def individual_node_complexity(self) -> Dict[str, int]:
+        # SELECT ( ROW_NUMBER()  OVER ( ORDER BY  SEQ8() ) -  1 ) * (step) + (start) AS id FROM ( TABLE (GENERATOR(ROWCOUNT => count)))
+        return {
+            PlanNodeCategory.WINDOW.value: 1,
+            PlanNodeCategory.ORDER_BY.value: 1,
+            PlanNodeCategory.LITERAL.value: 3,  # step, start, count
+            PlanNodeCategory.COLUMN.value: 1,  # id column
+            PlanNodeCategory.LOW_IMPACT.value: 2,  # ROW_NUMBER, GENERATOR
```

Comment:
shoudn't row_number and generator counted as function? since those are snowflake functions

Reply:
yep, we should. I'll fix it